Add LimitRanger plugin to labels, taints and annotations (#34939)
kubernetes/website: content/en/docs/reference/labels-annotations-taints/_index.md

This is useful if you want to target a specific namespace with a label
{{< glossary_tooltip text="selector" term_id="selector" >}}.

### kubernetes.io/limit-ranger

> Example: `kubernetes.io/limit-ranger: LimitRanger plugin set: cpu, memory request for container nginx; cpu, memory limit for container nginx`

Reviewer: For clarity:
```suggestion
Example: `kubernetes.io/limit-ranger: "LimitRanger plugin set: cpu, memory request for container nginx; cpu, memory limit for container nginx"`
```

> Used on: Namespace

Reviewer: ```suggestion
Used on: Pod
```

Kubernetes by default doesn't provide any resource limit, that means unless you explicitly define limits,
your container can consume unlimited CPU and memory.

> We can define default limit for pods by creating a LimitRange in the relevant namespace.

Reviewer: ```suggestion
You can define a default request or default limit for pods. You do this by creating a LimitRange
in the relevant namespace.
```

> Pods deployed after this LimitRange will have these limits applied to them.

Reviewer: ```suggestion
Pods deployed after you define a LimitRange will have these limits applied to them.
```

> The annotation `limit-ranger` indicates that limits were requested for the pod and they were applied successfully.

Reviewer: ```
The annotation `kubernetes.io/limit-ranger` indicates that limits were specified for the Pod,
and they were applied successfully.
```

> For more details - [LimitRange](/docs/concepts/policy/limit-range)

Reviewer: ```suggestion
For more details, read about [LimitRanges](/docs/concepts/policy/limit-range).
```

This is useful if you want to target a specific namespace with a label
{{< glossary_tooltip text="selector" term_id="selector" >}}.

### kubernetes.io/limit-ranger

Example: `kubernetes.io/limit-ranger: "LimitRanger plugin set: cpu, memory request for container nginx; cpu, memory limit for container nginx"`

Used on: Pod

Kubernetes by default doesn't provide any resource limit, that means unless you explicitly define limits,
your container can consume unlimited CPU and memory.
You can define a default request or default limit for pods. You do this by creating a LimitRange in the relevant namespace.
Pods deployed after you define a LimitRange will have these limits applied to them.
The annotation `kubernetes.io/limit-ranger` records that resource defaults were specified for the Pod,
and they were applied successfully.

> For more details, read about [LimitRanges](/docs/concepts/policy/limit-range).

Reviewer: Ah. I suspect that this annotation might also be applied on a PersistentVolumeClaim. If so, we ought to add that detail here before we merge it.

@SergeyKanzhelev do you have an opinion here?

Reviewer 2: yes, I noticed that PVC was added to the document as a quick fix. I'd prefer we address it as a separate PR.
